packages/angular_devkit/build_angular/src/builders/browser-rspack/webpack-factory-adapter.ts

```diff
@@ -0,0 +1,192 @@
+import path from 'node:path';
+import { RspackCLI } from '@rspack/cli';
+import { DevTool, Mode, rspack, RspackOptions } from '@rspack/core';
+import { from, Observable } from 'rxjs';
+import { NamedChunksPlugin } from '../../webpack/plugins/named-chunks-plugin';
+import { OccurrencesPlugin } from '../../webpack/plugins/occurrences-plugin';
+import { AngularWebpackPlugin } from '@ngtools/webpack';
+import webpack from 'webpack';
+import { Target } from '@rspack/core/dist/config/types';
+import { ProgressPlugin } from './plugins/progress-plugin';
+
+export function webpackFactory(options: any) {
+  return from(createWebpackFactoryFromRspackCLI(options)) as unknown as Observable<typeof webpack>;
+}
+
+async function createWebpackFactoryFromRspackCLI(options: any) {
```

Comment:
There should be something here to handle tsconfig paths. Similar to this: https://github.com/nrwl/nx-labs/blob/main/packages/rspack/src/utils/with-nx.ts#L20-L25

Rspack isn't handling this out of the box (neither does Webpack with ts-loader).